packages/dev/core/src/Rendering/IBLShadows/iblShadowsVoxelRenderer.ts

```diff
@@ -394,7 +417,7 @@ export class _IblShadowsVoxelRenderer {
         for (let mipIdx = 1; mipIdx <= this._mipArray.length; mipIdx++) {
             const mipDim = this._voxelResolution >> mipIdx;
             const mipSize: TextureSize = { width: mipDim, height: mipDim, depth: mipDim };
-            this._mipArray[mipIdx - 1] = new ProceduralTexture("voxelMip" + mipIdx, mipSize, "generateVoxelMip", this._scene, voxelAxisOptions, false);
+            this._mipArray[mipIdx - 1] = new ProceduralTexture("voxelMip" + mipIdx, mipSize, "iblGenerateVoxelMip", this._scene, voxelAxisOptions, false);

```

Comment:
@deltakosh this doesn't look like it's loading a WebGPU-specific shader.

Reply:
Nice catch! thanks a ton!